functions/src/lobby.ts

```diff
@@ -0,0 +1,56 @@
+import * as functions from "firebase-functions";
+import { lobbyCollection, userCollection } from "./firestore-collections";
+import { Lobby } from "./firestore-types/lobby";
+import { UserData } from "./firestore-types/users";
+
+export const startGame = functions.https.onCall(async (data: { code: string }, context) => {
```

Comment:
we should probobly makes types for the imnput data and share the types between the server and the client so that if we want to change it in the future it will force us to change it in both places